src/ripple/app/tests/HashRouter_test.cpp

```diff
@@ -28,9 +28,10 @@ namespace test {
 class HashRouter_test : public beast::unit_test::suite
 {
     void
-    testExpiration()
+    testNonExpiration()
```

Comment:
@ximinez This does test what I want thanks. However, how about just a simple test that sets two keys, access one of them on the next tick to touch it, then on the next tick see that the non-accessed key expired and the accessed key did not (in code: https://github.com/seelabs/rippled/commit/df4da2fbf6a613f91366cce9b742a7e0a4d2ed54)

It's easier for me to understand the change of behaviour in the "simple" test.